_clients/opensearch-py-ml.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: Opensearch-py-ml
+nav_order: 11
+---
+
+`opensearch-py-ml` is a Python client that provides a suite of data analytics and machine learning (ML) tools for OpenSearch. It provides data analysts with the following abilities:
+
+- Call OpenSearch indexes and manipulate each index as a [pandas.DataFrame](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html). These data frames support complex filtering and aggregation operations.
```

Comment:
```eland.DataFrame wraps an Elasticsearch index in a Pandas-like API and defers all processing and filtering of data to Elasticsearch instead of your local machine. This means you can process large amounts of data within Elasticsearch from a Jupyter Notebook without overloading your machine.```

This is from eland. We are also doing the same (as we forked from eland). Maybe we can rephrase this?